app/views/pages/how-platformos-works/platformos-workflow/development-workflow.liquid

```diff
@@ -8,41 +8,62 @@ slug: how-platformos-works/platformos-workflow/development-workflow
 
-You can choose other version control systems, or decide to go without any: all that is required is your codebase and an Instance you can sync/deploy to.
+What you’ll need:
 
-## Development Instances
+* [Access to the Partner Portal](/get-started/accessing-partner-portal)
 
-Development Instances are different copies of the same Instance/codebase used for developing, updating, and testing a site without risking the integrity of the live site.
+### 2. Create codebase
+In order to correctly communicate with the platformOS engine and API, your codebase should be organized into a [specific directory structure](/how-platformos-works/platformos-workflow/codebase). To create your codebase in a local environment follow the steps in the [Creating the Codebase](/get-started/creating-codebase) topic.
 
-You can set up different types of Instances (staging/production) through the Partner Portal.
+What you’ll need:
+
+* Node.js (for [Mac](/get-started/installing-nodejs-mac) or [Windows](/get-started/installing-nodejs-windows))
+* [pos-cli](/get-started/installing-pos-cli)
+
+### 3. Setup pos-cli
+The **[pos-cli](https://github.com/mdyd-dev/pos-cli)** is a command line interface that helps you deploy configuration files and assets to your platformOS site. It also allows you to use team collaboration tools like git in your workflow, and it works with cloud storage providers like [Amazon AWS S3](https://aws.amazon.com/s3/).
+
+You can use the pos-cli to deploy or sync your codebase to the Instance as described in the [Deploying and Syncing Changes](/get-started/deploying-syncing-changes) tutorial.
+
+What you’ll need:
 
-### Local codebase
-You develop your site (write code) in your codebase in your local environment.
+* [pos-cli](/get-started/installing-pos-cli)
 
-The framework for the codebase of the Instance is your codebase organized into the required directory structure. To learn more about the directories and files in your platformOS codebase, see [platformOS Components](/how-platformos-works/platformos-workflow).
+## Development
 
-### Staging Instance
-The staging Instance can be used for testing, presenting the changes to the client for approval, QA (Quality Assurance), or UAT (User Acceptance Testing).
+We recommend using a **version control system**. Internally, we use [GitHub](https://github.com/) to host and collaborate on our [git](https://git-scm.com/about) repositories. You can choose other version control systems, or decide to go without any: all that is required is your codebase and an Instance you can sync/deploy to.
 
-### Production Instance
+This is a diagram of our development workflow:
+
+<img src="{{ 'images/workflow/development_workflow.png' | asset_url }}" alt="Feedback block" />
+
+### Development Instances
+
+Development Instances are different copies of the same Instance/codebase used for developing, updating, and testing a site without risking the integrity of the live site.
```

Comment:
Well, they are not copies, otherwise there would be no point in having them.

They are used for different purposes and different risk is lied on them.

Development - anything can be broken because dev is working on it constantly. Should be used for development.
Staging - should not be broken, only working things should be merged there. Should be used for QA, UAT, etc.
Production - once agreed upon that everything works fine on staging, changes are promoted to production (periodically, or automatically after tests on staging pass) - Should be used by end users.

Reply:
Heh, i see explanation below, so ignore my explanation, but the point about copy is still valid

Reply:
Thank you!